app/views/qsets/show.html.erb

```diff
@@ -24,50 +24,56 @@
                 </div>
-              </div>
-              <div class="col-sm-6">
-                <div style="cursor:default" class="question-content">
-                  <%= link_to question_path(question) do%>
-                  <%= question.text %>
-                  <% end %>
-                  <br/>
-                  <div class="pull-right">
-                      <!-- Creates the edit and delete options for each question -->
-                      <% if can? :manage, question %>
-
-                        <%= link_to edit_question_path(question), class:'btn shortcut-button-link' do%>
-                          <span class="glyphicon glyphicon-edit shortcut-button-glyphicon"></span>
+                <div class="col-xs-8 col-sm-9">
+                  <div class="question-content">
+                    <div class="question-content-inner">
+                      <span>
+                        <%= link_to question_path(question) do%>
+                        <%= question.text %>
                         <% end %>
+                      </span>
+                    </div>
+                    <br/>
+                    <div class="pull-right">
+                        <!-- Creates the edit and delete options for each question -->
+                        <% if can? :manage, question %>
 
-                        <%= link_to question_path(question), method: :delete, class:'btn shortcut-button-link' do%>
-                          <span class="glyphicon glyphicon-trash shortcut-button-glyphicon"></span>
-                        <% end %>
+                          <%= link_to edit_question_path(question), class:'btn shortcut-button-link' do%>
+                            <span class="glyphicon glyphicon-edit shortcut-button-glyphicon"></span>
+                          <% end %>
 
-                      <% end %>
+                          <%= link_to question_path(question), method: :delete, class:'btn shortcut-button-link' do%>
+                            <span class="glyphicon glyphicon-trash shortcut-button-glyphicon"></span>
+                          <% end %>
+
+                        <% end %>
+                    </div>
                   </div>
                 </div>
               </div>
-              <div class="col-sm-2">
-                <a href=""><%= question.user.full_name %></a>
-              </div>
-              <div class="col-sm-2">
-                  <a data-toggle="modal"
-                    href="#"
-                    data-target="#question_display_Modal"
-                    data-answer="<%= question.answers[0].text %>"
-                    data-qid="<%= question.id %>"
-                    data-question="<%= question.text %>"
-                    class="btn btn-primary">
-                      Quiz Me
-                  </a>
+              <div class="col-xs-3">
+                <div class="col-sm-5">
+                  <%= link_to question.user.full_name, user_path(question.user.id) %>
```

Comment:
I think you can get even shorter/more magical/more 'Rails-like' by just using `user_path(question.user)`, but this is fine. Thanks.